cores/migrations/20240725014146_FixCategoryRelations.up.sql

```diff
@@ -0,0 +1,3 @@
+-- Add migration script here
+ALTER TABLE categories
+DROP COLUMN markdown_id;
```

Comment:
We have the markdown_categories join table that associates a markdown with multiple categories so we don't want this direction FK too